src/dotnet/APIView/APIViewWeb/Managers/NotificationManager.cs

```diff
@@ -60,20 +69,17 @@ public async Task NotifyApproversOfReview(ClaimsPrincipal user, string reviewId,
             {
                 var reviewerProfile = await _userProfileRepository.TryGetUserProfileAsync(reviewer);
                 await SendUserEmailsAsync(review, reviewerProfile,
-                    GetApproverReviewContentHeading(userProfile, false),
```

Comment:
Are we also removing some notification emails as part of this change?

Reply:
Or do we no longer need the plain text with the new approach?

Reply:
We no longer need that. SendGrid, for some reasons, require plain text.

Reply:
Ok so if that value is set we have to restart the server to unset it?

Reply:
Opps that question was intended for the test email address configuration